packages/core/src/model.ts

```diff
@@ -22,13 +24,15 @@ export namespace Field {
   export const boolean: Type[] = ['boolean']
   export const date: Type[] = ['timestamp', 'date', 'time']
   export const object: Type[] = ['list', 'json']
+  export const primary: Type[] = ['primary']
 
   export type Type<T = any> =
     | T extends number ? 'integer' | 'unsigned' | 'float' | 'double' | 'decimal'
     : T extends string ? 'char' | 'string' | 'text'
     : T extends boolean ? 'boolean'
     : T extends Date ? 'timestamp' | 'date' | 'time'
     : T extends unknown[] ? 'list' | 'json'
+    : T extends Primary ? 'primary' | 'unsigned'
```

Comment:
why `unsigned` here?

Reply:
This allows mongo users to only change type `Primary` and still use old style autoInc as kind of compatibility.